vendor/bat-native-ads/include/bat/ads/ads.h

```diff
@@ -206,7 +211,8 @@ class ADS_EXPORT Ads {
 
   // Should be called to get ads history. Returns |std::map<uint64_t,
   // std::vector<AdsHistory>>| in the format |{timestamp, array<AdsHistory>}|
-  virtual std::map<uint64_t, std::vector<AdsHistory>> GetAdsHistory() = 0;
+  virtual std::map<uint64_t, std::vector<AdsHistory>> GetAdsHistory(
+      const AdHistoryFilterType adHistoryFilterType) = 0;

```

Comment:
`adHistoryFilterType` should use `snake_case`

Reply:
Done